tests/modules/segmentation/qc_utils/conftest.py

```diff
@@ -0,0 +1,88 @@
+import pytest
+import pathlib
+import numpy as np
+import json
+import PIL.Image
+import networkx as nx
+from itertools import combinations, product
+
+from ophys_etl.types import ExtractROI
+
+
+@pytest.fixture(scope='session')
+def list_of_roi():
```

Comment:
Not required, but I tend to like to append `_fixture` to any pytest fixture names to make it explicit that a parameter to a test is not some weird function, but a fixture which usually returns a data type.

`def list_of_roi_fixture()`